python/example/utils.py

```diff
@@ -28,36 +29,6 @@ def to_array(v, size):
     return v
 
 
-def make_grid(size, lower, upper):
-    assert lower.shape[0] == upper.shape[0]
-
-    # make initial 1d grid
-    theta_grids = (
-        Gridder.make_grid(size, lower[i], upper[i]) for i in range(len(lower))
-    )
-    # make corresponding radius
-    radius = [Gridder.radius(size, lower[i], upper[i]) for i in range(len(lower))]
-    return theta_grids, radius
-
-
-def make_cartesian_grid_range(size, lower, upper, grid_sim_size):
```

Comment:
I moved this function into pykevlar.grid since it seemed useful to have as part of the library. Currently all of our examples are using this function!